src/main/kotlin/org/jitsi/nlj/transform/node/outgoing/RetransmissionSender.kt

```diff
@@ -54,37 +50,16 @@ class RetransmissionSender(
     private var numRetransmittedPlainPackets = 0
 
     init {
-        val rtxPayloadTypeFilter = object : MapEventValueFilter<Byte, PayloadType>({ it is RtxPayloadType }) {
-            override fun entryAdded(key: Byte, value: PayloadType) =
-                setRtxPayloadTypeAssociation(value as RtxPayloadType)
-
-            override fun entryUpdated(key: Byte, newValue: PayloadType) =
-                setRtxPayloadTypeAssociation(newValue as RtxPayloadType)
-
-            override fun entryRemoved(key: Byte) {
-                associatedPayloadTypes.remove(key.toInt())
-            }
-        }
-        streamInformationStore.onRtpPayloadTypeEvent(rtxPayloadTypeFilter.mapEventHandler)
-    }
-
-    private fun setRtxPayloadTypeAssociation(rtxPayloadType: RtxPayloadType) {
-        rtxPayloadType.associatedPayloadType?.let { associatedPayloadType ->
-            associatedPayloadTypes[rtxPayloadType.pt.toInt()] = associatedPayloadType
-            logger.cdebug { "Associating RTX payload type ${rtxPayloadType.pt.toInt()} " +
-                "with primary $associatedPayloadType" }
-        } ?: run {
-            logger.cerror { "Unable to parse RTX associated payload type from payload " +
-                "type $rtxPayloadType" }
-        }
+        streamInformationStore.onRtpPayloadTypeEvent(rtxPayloadTypeStore.mapEventHandler)
     }
 
     override fun transform(packetInfo: PacketInfo): PacketInfo? {
         val rtpPacket = packetInfo.packetAs<RtpPacket>()
         numRetransmissionsRequested++
         // note(george) this instance gets notified about both remote/local ssrcs (see Transeiver.addSsrcAssociation)
         // so, in the case of firefox, we end up having rtx (associated) ssrcs but no rtx (associated) payload type.
-        val rtxPt = associatedPayloadTypes[rtpPacket.payloadType.toPositiveInt()] ?: return retransmitPlain(packetInfo)
+        val rtxPt =
+            rtxPayloadTypeStore.getOriginalPayloadType(rtpPacket.payloadType) ?: return retransmitPlain(packetInfo)
```

Comment:
I just noticed the variable name and checked the code: the original map here maps a media PT to it's associated RTX PT. But the RTX store has the opposite mapping.

Not related to this PR, but a general argument about having more documentation: I had to trace the usage of `RetransmissionSender` to figure out what `transform` is supposed to do. It would have been better if this was in the docs.

Reply:
>  just noticed the variable name and checked the code: the original map here maps a media PT to it's associated RTX PT. But the RTX store has the opposite mapping.

Nice catch!  This one can't be verified using the harnesses (because we don't send probing there) so i just checked the logged mappings and didn't even notice.  I think I'll add a small test for this.

> Not related to this PR, but a general argument about having more documentation: I had to trace the usage of RetransmissionSender to figure out what transform is supposed to do. It would have been better if this was in the docs.

I'll add something while I'm in there.

Reply:
So it looks like I trusted the existing documentation (which was wrong) and didn't check the code and this lead to more confusion. I don't know what that says about the idea of requiring more documentation. When the docs are wrong, they are worse than useless.